docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | Warm up ratio                                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+To train the model from scratch, run:
+
+.. code::
+
+      python examples/nlp/token_classification/punctuation_and_capitalization_train.py \
+             model.dataset.data_dir=<PATH/TO/DATA_DIR> \
+             trainer.gpus=[0,1] \
+             optim.name=adam \
+             optim.lr=0.0001 \
+             model.nemo_path=<PATH/TO/SAVE/.nemo>
+
+To train from the pre-trained model, use:
+
+.. code::
+
+      python examples/nlp/token_classification/punctuation_and_capitalization_train.py \
+             model.dataset.data_dir=<PATH/TO/DATA_DIR> \
+             pretrained_model=<PATH/TO/SAVE/.nemo>
+
+
+Required Arguments for Training
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :code:`model.dataset.data_dir`: Path to the `data_dir` with the pre-processed data files.
+
+Optional Arguments
+^^^^^^^^^^^^^^^^^^
+* :code:`pretrained_model`: pretrained PunctuationCapitalization model from list_available_models() or path to a .nemo file, for example: Punctuation_Capitalization_with_BERT_base_uncased or your_model.nemo
+* :code:`--config-name`: Path to the config file to use. The default config file for the model is `/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml`. You may update the config file from the file directly. The other option is to set another config file via command line arguments by :code:`--config-name=<CONFIG/FILE/PATH>`. For more details about the config files and different ways of model restoration, see tutorials/00_NeMo_Primer.ipynb
+* Other arguments to override fields in the specification file, please see the note below.
+
+.. note::
+
+    All parameters defined in the configuration file could be changed with command arguments. \
+    For example, the sample config file mentioned above has :code:`validation_ds.batch_size` set to 64. \
+    However, if you see that the GPU utilization can be optimized further by using a larger batch size, \
+    you may override to the desired value, by adding the field :code:`validation_ds.batch_size=128` over the command line.
+    You may repeat this with any of the parameters defined in the sample configuration file.
+
+
+
+Important parameters
+^^^^^^^^^^^^^^^^^^^^
+
+Below is the list of parameters could help improve the model:
+
+- language model (`model.language_model.pretrained_model_name`)
```

Comment:
We need to have a page for supported model and description of language_model for all nlp models to avoid duplications. The same for optimizer. @ericharper